lib/route/link.c

```diff
@@ -2823,6 +2838,17 @@ pid_t rtnl_link_get_ns_pid(struct rtnl_link *link)
 	return link->l_ns_pid;
 }
 
+/**
+ * Return permanent hardware address of link object
+ * @arg link		Link object
+ *
+ * @return Permanent hardware address or NULL if not set.
+ */
+struct nl_data *rtnl_link_get_perm_addr(struct rtnl_link *link)
```

Comment:
compare to `rtnl_link_get_addr`. That one returns a `struct nl_addr` type. I think that should also be used for the perm-addr, or why not?